doc/extending/creating_an_op.rst

```diff
@@ -165,35 +151,6 @@ or :meth:`Op.make_thunk`.
   :meth:`COp.c_code` and other related ``c_**`` methods. Note that an
   :class:`Op` can provide both Python and C implementations.
 
-  :meth:`Op.make_thunk` method is another alternative to :meth:`Op.perform`.
```

Comment:
This is outdated, and tbh I don't think it's necessary for 99.9999% of the people reading this page